voc/python/ast.py

```diff
@@ -986,27 +995,71 @@ def _create_function(self, node, func_name, decorator_list):
 
         # Now actually define the function.
         for decorator in decorator_list:
-            self.visit(decorator)
-            self.context.add_opcodes(
-                JavaOpcodes.CHECKCAST('org/python/Callable'),
-                java.Array(1),
-                JavaOpcodes.DUP(),
-                JavaOpcodes.ICONST_0(),
-            )
+            # The @super decorator on __init__() is a special case. It tells us how
```

Comment:
Makes sense. The code won't be re-used anywhere, but it would certainly help with readability.